lineman/app/components/group_page/group_page.scss

```diff
@@ -59,12 +59,23 @@
   @include card;
 }
 
+.group-page__threads {
+  padding: 10px 0 17px;
+  h2 {
+    display: inline-block;
+    margin: 0 17px 17px 17px;
+    padding-top: 7px;
+  }
+}
+
 .group-page__new-thread{
-  float: right;
-  margin-top: -5px;
-  margin-bottom: 10px;
-  .btn { @include fontSmall; }
+  .btn {
```

Comment:
As with the headers above I have tried this naming convention and it doesn't work.  Feel free to try and refactor it yourself.   I have tested this PR in 3 browsers and just about every mobile device, if you make changes please retest.